docs/user/admin.rst

```diff
@@ -75,12 +75,14 @@ Kolibri users can have different roles with respective access to features:
   * View content and have their progress tracked
   * View *Coach Reports* to track progress of other users and usage stats for individual exercises
   * Create/Edit/Delete other **Admins** and **Learners**
+  * Create/Edit/Delete *Classes* and *Groups* and enroll users in them
```

Comment:
I agree that we should establish some convention, and the challenging thing about both rST and Markdown is that they do not leave much room for distinguishing concepts with styling (compared to HTML) because all you have to play with are **bold** and *italic*.